pkg/controller/core.oam.dev/v1alpha2/core/scopes/healthscope/healthscope_controller.go

```diff
@@ -361,3 +367,87 @@ func (r *Reconciler) UpdateStatus(ctx context.Context, hs *v1alpha2.HealthScope,
 		return r.client.Status().Update(ctx, hs, opts...)
 	})
 }
+
+func (r *Reconciler) patchHealthStatusToApplications(ctx context.Context, appCs []*AppHealthCondition, hs *v1alpha2.HealthScope) error {
```

Comment:
Why is the parameter of `[]*AppHealthCondition` shorten to `appCs`? It seems I didn't see this abbreviation style:)

Reply:
revised